.github/workflows/windows-tests.yaml

```diff
@@ -1,9 +1,14 @@
-name: Windows tests
+name: Windows Tests
 
 on:
-  workflow_dispatch: {}
-  schedule:
-    - cron: '0 16 * * *'  # every day at 4 p.m. UTC / 9 a.m. PDT
+  workflow_dispatch:
```

Comment:
```suggestion
  workflow_dispatch: {}
```

Reply:
it should work either way but updated since `permissions` is the same way and it's easier to read perhaps 👍